File: old/fizzbuzz/src/c++/fizzbuzz.cpp
#include <iostream>

using std::cout, std::endl;

int main() {
    for (auto n = 1; n <= 50; n++) {
        if (n % 3 == 0 && n % 5 == 0) {
            cout << "fizz buzz" << endl;
        } else if (n % 3 == 0) {
            cout << "fizz" << endl;
        } else if (n % 5 == 0) {
            cout << "buzz" << endl;
        } else {
            cout << n << endl;
        }
    }
}

File: old/fizzbuzz/src/elixir/fizzbuzz.exs
for n <- 1..50 do
  cond do
    Integer.mod(n, 3) == 0 and Integer.mod(n, 5) == 0 -> IO.puts "fizz buzz"
    Integer.mod(n, 3) == 0 -> IO.puts "fizz"
    Integer.mod(n, 5) == 0 -> IO.puts "buzz"
    :else -> IO.puts n
  end
end
